### PR TITLE
Fix verify-all

### PR DESCRIPTION
**Pull request by chuckha**
Signed-off-by: Chuck Ha <chuckh@vmware.com>

/assign @neolit123

### CODE REVIEW

**Comment by chuckha** [comment]
/hold

no that's not right either.

**Comment by chuckha** [comment]
/hold cancel

fixed

File: hack/verify-all.sh [hunk 1]
```diff
@@ -37,64 +37,64 @@ res=0
 
 if [[ "${VERIFY_WHITESPACE:-true}" == "true" ]]; then
   echo "[*] Verifying whitespace..."
-  hack/verify-whitespace.sh || res=1
-  failure $res "verify-whitespace.sh"
+  hack/verify-whitespace.sh
+  failure $? "verify-whitespace.sh"
```
Comment (neolit123) [inline hack/verify-all.sh]: hm, maybe i'm missing something. can you explain what the fix is here?
each `verify-*.sh` has a `|| res=1` which is fine and `failure()` only notifies in this case.
then near EOF `res=1/0` triggers an overall summary and the exit code matches `res`.

Comment (neolit123) [reply hack/verify-all.sh]: ok, i see.
`$res` can leak from previously failed `verify-` calls.

**Pull request merged**